doc/releases/changelog-0.9.0.md

```diff
@@ -50,9 +30,12 @@
   >>> circuit()
   (Array([[0], [1], [0], [1], [1]], dtype=int64),
   Array([[0], [1], [1], [0], [1]], dtype=int64),
-  Array([[1], [0], [1], [1], [0], [1],[0]], dtype=int64))
+  Array([[1], [0], [1], [1], [0], [1], [0]], dtype=int64))
   ```
 
+  Note that other measurement types, such as `expval` and `probs`, currently
+  do not support shot-vectors.
```

Comment:
Note: we should make an issue about adding (_efficient_ -- that is, there should only be one circuit execution) support for shot vectors to other measurement processes.